src/jit/compiler.h

```diff
@@ -3059,6 +3059,7 @@ class Compiler
     GenTreePtr impTreeList; // Trees for the BB being imported
     GenTreePtr impTreeLast; // The last tree for the current BB
 
+public:
```

Comment:
Is this some unrelated change?

Reply:
I need access to `CHECK_SPILL_ALL`, it was strange to have this enum private, because functions, that accept it are public.

Reply:
You mean `CHECK_SPILL_NONE`?

There is another `public: ` just below this one, so maybe you can get rid of the one that follows?

Reply:
Done, I think  we can name this enum and replace `unsigned chkLevel` with it in another PR.